packages/app-desktop/gui/Sidebar/Sidebar.tsx

```diff
@@ -502,7 +516,8 @@ const SidebarComponent = (props: Props) => {
 					isSpecialItem={true}
 					href="#"
 					selected={selected}
-					onClick={onAllNotesClick_}
+					onClick={() => { onAllNotesClick_(); folderItem_click(ALL_NOTES_FILTER_ID); }}
```

Comment:
Actually I don't see why there are calls to two different functions here. `onAllNotesClick_()` should contain all the required behaviour, so maybe move `folderItem_click(ALL_NOTES_FILTER_ID)` to that function?